README.md

```diff
@@ -1,29 +1,92 @@
-# Tuples 
+Learn Tuples with Swift
+======================
 
+We've come a long way to learn the fundamentals of Swift. With our knowledge of variables, `Strings`, `Booleans` and collections, we can now create *functions*.  Functions are the building blocks of any application - large or small. To introduce the new (and funny-sounding) idea of `tuples`, let's review how functions can be used to send and receive data.
 
-## Learning Objectives (Students will be able to ....) 
-- Describe a tuple as being a type that groups multiple values into a single compound value.
-- Explain that values within a tuple can be of any type and do not have to be of the same type as each other.
-- Iterate over the key-value pairs in a dictionary where each item is returned as a (key, value) tuple.
-- Create a tuple and name the individual elements
 
-## What the Student Already Knows
-A student should already understand the concept of variables, constants, Int, String, Bool, Array, Dictionary and functions but not Tuples.
+A Game of Chance
+----------------
+ Suppose you were creating a game that included the use of dice. You will randomly roll a pair a dice to receive some prize or to advance in the game. With our current knowledge of functions, the actions to start the game may look something like this:
 
-## Outline/ Notes
 
-* Start with the known- students know how to iterate over arrays. Show an example of iterating over an array. 
-    * Present some real world problem such as, “If you have a dictionary where the keys represent the seasons and the values represent your favorite color associated with that season, is there an easy way to print out a list that for each sentence and color that says “In the {season} my favorite color is {color}.”
-    - If you have a better example feel free to use it, but always aim for a real world example. And you can use that example throughout the rest of your reading and/or use other new examples. 
-    * The answer is tuples and this sets the stage for writing the content that covers your learning objectives above.--- You can iterate over the key-value pairs in a dictionary with a for-in loop. Each item in the dictionary is returned as a (key, value) tuple. Explain how you can decompose the tuple’s members into temporary constants as part of the iteration.
+```swift
+
+func startGame() {
+    let dice: Int = throwDice()
+    print("Congrats - you rolled \(dice)!")
+}
```

Comment:
Where the student will understand what functions are, the following line of code might shake their confidence a little bit before they are shown the ultimate function that will answer the question above which is.. "How can we create a function that returns multiple types into one compound value"?
```
let dice: Int = throwDice()
```
That being said, in reading further you expanded upon this which is perfect! I would prefer if we explained the startGame function from start to finish in a paragraph. That way, the student can properly see how to structure a sentence in explaining what it is that's going on here.

For example.. startGame is a function that takes no arguments and doesn't have a return type. dice is a constant (where the value can't change) of type Int. We are not explicitly assigning it a value of 1, 2 or 3.. we are calling on another function throwDice which must return back to us an Int.